aten/src/TH/THTensor.cpp

```diff
@@ -25,47 +25,8 @@ void THTensor_free(THTensor *self)
 }
 
 void THTensor_setStorage(THTensor *self, THStorage *storage_, ptrdiff_t storageOffset_, at::IntArrayRef size_, at::IntArrayRef stride_) {
```

Comment:
So after this PR, who calls this (resp. the cuda version)? Or is it part of the public API and never going away?

Reply:
good question.  It's all internal calls, i.e. things calling `THTensor_(set)`, `THTensor_(newWithStorage1d)`.  Most of those look easy to remove except that the copy code uses `THTensor_(newWithStorage1d)` and that's a bit hairy.

Reply:
Thanks for the info - yeah just looked at the callsites of `newWithStorageId` and they look nontrivial to excise :)